protocols/identify/src/protocol.rs

```diff
@@ -52,15 +52,26 @@ pub struct RemoteInfo {
     _priv: ()
 }
 
-/// Object used to send back information to the client.
-pub struct IdentifySender<T> {
+/// The substream on which a reply is expected to be sent.
+pub struct ReplySubstream<T> {
     inner: Framed<T, codec::UviBytes<Vec<u8>>>,
 }
 
-impl<T> IdentifySender<T> where T: AsyncWrite {
-    /// Sends back information to the client. Returns a future that is signalled whenever the
-    /// info have been sent.
-    pub fn send(self, info: IdentifyInfo, observed_addr: &Multiaddr) -> IdentifySenderFuture<T> {
+impl<T> fmt::Debug for ReplySubstream<T> {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        write!(f, "identify::protocol::ReplySubstream")
```

Comment:
```suggestion
        f.debug_tuple("ReplySubstream").finish()
```

There are methods on `f` to write out debug implementations. Also the naming scheme is generally just the name of the struct.